source/pages/albedo.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    filt = daily_albedo &gt; 0.6
</pre><p>Let's just call the variable <code>snow_filter</code> or <code>snow_bool</code> and not be lazy ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>